src/libs/actions/IOU.ts

```diff
@@ -4085,6 +4085,12 @@ function requestMoney(requestMoneyInformation: RequestMoneyInformation) {
 
     InteractionManager.runAfterInteractions(() => removeDraftTransaction(CONST.IOU.OPTIMISTIC_TRANSACTION_ID));
     Navigation.dismissModal(isSearchTopmostCentralPane() ? undefined : activeReportID);
+
+    const trackReport = Navigation.getPreviousTrackReport(linkedTrackedExpenseReportAction?.childReportID);
+    if (trackReport?.key) {
+        Navigation.isNavigationReady().then(() => Navigation.removeScreenByKey(trackReport.key));
```

Comment:
Can we move `.isNavigationReady()` to `removeScreenByKey` function itself? That way we won't have to call it each time manually

Reply:
But we're still calling `isNavigationReady` here. Is it needed (in that case we can move it to `removeScreenByKey`) or is it not needed (in that case we can remove it from here)?